pandas/tests/test_lib.py

```diff
@@ -39,6 +40,11 @@ def test_fast_unique_multiple_list_gen_sort(self):
         out = lib.fast_unique_multiple_list_gen(gen, sort=False)
         tm.assert_numpy_array_equal(np.array(out), expected)
 
+    def test_fast_unique_multiple_unsortable_runtimewarning(self):
+        arr = [np.array(["foo", pd.Timestamp("2000")])]
+        with tm.assert_produces_warning(RuntimeWarning):
+            lib.fast_unique_multiple(arr, sort=None)
```

Comment:
Can you test this from an end user perspective?

Reply:
see above